hack/e2e/kops/kops.sh

```diff
@@ -32,6 +32,7 @@ function kops_create_cluster() {
   KOPS_PATCH_FILE=${10}
   KOPS_PATCH_NODE_FILE=${11}
   KOPS_STATE_FILE=${12}
+  export KOPS_RUN_TOO_NEW_VERSION=true
```

Comment:
If necessary, can we export this only once at the top of the file? 

Even better, if this parameter is only relevant during cluster bootstrap then lets export it in `kops_create_cluster()` and remove the other instances.

Reply:
This used to be once at top of file, and was set when the other script sourced kops.shm but @ConnorJC3 preferred in each function: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2160#discussion_r1799892385

This parameter is relevant to any operation involving a too-new k8s cluster, which is why it's not just in `kops_create_cluster`.

For the record, I too would prefer it just once at top of file, but don't think it's worth holding up the PR.

Reply:
Changed to function scoped `declare -x KOPS_TOO_NEW_VERSION=true` that @ConnorJC3 found as compromise.  (Though it is bash voodoo)